.env

```diff
@@ -0,0 +1,7 @@
+CASDOOR_ENDPOINT=http://172.22.105.143:8008
```

Comment:
Use the same Casdoor config as: https://github.com/casdoor/casdoor-python-vue-sdk-example?tab=readme-ov-file#configuration